src/docs/markdown/caddyfile/directives/log.md

```diff
@@ -253,7 +314,7 @@ log {
 
 Use Common Log Format (CLF):
 
-<span class="warning">⚠️ The `single_field` format is deprecated and will be removed in a future version. To encode logs in common log format, please use the [`format-encoder`](https://github.com/caddyserver/format-encoder) plugin.</span>
+<span class="warning">⚠️ The `single_field` format is deprecated and removed in Caddy v2.5. To encode logs in common log format, please use the [`format-encoder`](https://github.com/caddyserver/format-encoder) plugin.</span>
```

Comment:
Maybe we should just remove references to single_field and instead help people find what they need in the format-encoder directly.